FordLabs/peoplemover
# Migrate cypress tests from cypress 6.5.0 to 9.6.0

## Issue
Upgraded cypress from 6.5.0 to 9.6.0 and updated tests accordingly

## What was done
- [x] Upgraded cypress from 6.5.0 to 9.6.0
- [x] Updated moment instances in cypress tests from using `Cypress.moment` which is depreciated, to using moment directly
- [x] Added `{ force: true }` to tests typing into input fields that are using an outside library. Some of the fields do not use a visible html input field, so you have to force type for cypress to actually type into the field.  

## How to test
1. Run cypress tests and ensure they are running the latest version of cypress and that all tests pass.
